TheCodeBlog/SceneDelegate.swift

```diff
@@ -62,3 +62,2156 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
 }
 
+// // MARK: - 10 - Dispatch Async
+
+// func checkDispatchSyncIsNotCalledOnMain() {
+//   let excludedFiles = [
+//     "Dangerfile"
+//   ]
+//   for file in filter(files: filesChanged, with: [.swift]) {
+//     guard excludedFiles.allSatisfy({ !file.contains($0) }) else { continue }
+//     let fileLines = read(file: file, danger: danger)
+
+//     for (index, line) in fileLines.enumerated() {
+//       if line.contains("DispatchQueue.main.sync") {
+//         let link = "https://stackoverflow.com/questions/44324595/difference-between-dispatchqueue-main-async-and-dispatchqueue-main-sync"
+//         warn(message: "Please async on the main queue. [More information](\(link))",
+//              file: file,
+//              line: index + 1)
+//       }
+//     }
+//   }
+// }
+
+
+// // MARK: - 10 - Dispatch Async
+
+// func checkDispatchSyncIsNotCalledOnMain() {
+//   let excludedFiles = [
+//     "Dangerfile"
+//   ]
+//   for file in filter(files: filesChanged, with: [.swift]) {
+//     guard excludedFiles.allSatisfy({ !file.contains($0) }) else { continue }
+//     let fileLines = read(file: file, danger: danger)
+
+//     for (index, line) in fileLines.enumerated() {
+//       if line.contains("DispatchQueue.main.sync") {
+//         let link = "https://stackoverflow.com/questions/44324595/difference-between-dispatchqueue-main-async-and-dispatchqueue-main-sync"
+//         warn(message: "Please async on the main queue. [More information](\(link))",
+//              file: file,
+//              line: index + 1)
+//       }
+//     }
+//   }
+// }
+
+
```

Comment:
<!--
  1 failure:  Limit vertical wh...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: TheCodeBlog/SceneDelegate.swift;
  Line: 108;
-->
- :no_entry_sign: Limit vertical whitespace to a single empty line. Currently 2. (`vertical_whitespace`)